modules/ztp-checking-du-cluster-config.adoc

```diff
@@ -0,0 +1,518 @@
+.Example output
+[source,terminal]
+----
+sending: GET PORT_DATA_SET
+  3cecef.fffe.7a7020-1 seq 0 RESPONSE MANAGEMENT PORT_DATA_SET
+    portIdentity            3cecef.fffe.7a7020-1
+    portState               SLAVE
+    logMinDelayReqInterval  -4
+    peerMeanPathDelay       0
+    logAnnounceInterval     1
+    announceReceiptTimeout  3
+    logSyncInterval         0
+    delayMechanism          1
+    logMinPdelayReqInterval 0
+    versionNumber           2
+  3cecef.fffe.7a7020-2 seq 0 RESPONSE MANAGEMENT PORT_DATA_SET
+    portIdentity            3cecef.fffe.7a7020-2
+    portState               LISTENING
+    logMinDelayReqInterval  0
+    peerMeanPathDelay       0
+    logAnnounceInterval     1
+    announceReceiptTimeout  3
+    logSyncInterval         0
+    delayMechanism          1
+    logMinPdelayReqInterval 0
+    versionNumber           2
+----
+
+.. Run the following `pmc` command to check the PTP clock status:
++
+[source,terminal]
+----
+$ oc -n openshift-ptp rsh -c linuxptp-daemon-container ${PTP_POD_NAME} pmc -u -f /var/run/ptp4l.0.config -b 0 'GET TIME_STATUS_NP'
+----
++
+.Example output
+[source,terminal]
+----
+sending: GET TIME_STATUS_NP
+  3cecef.fffe.7a7020-0 seq 0 RESPONSE MANAGEMENT TIME_STATUS_NP
+    master_offset              10 <1>
+    ingress_time               1657275432697400530
+    cumulativeScaledRateOffset +0.000000000
+    scaledLastGmPhaseChange    0
+    gmTimeBaseIndicator        0
+    lastGmPhaseChange          0x0000'0000000000000000.0000
+    gmPresent                  true <2>
+    gmIdentity                 3c2c30.ffff.670e00
+----
+<1> `master_offset` should be between -100 and 100 ns.
+<2> Indicates that the PTP clock is synchronized to a master, and the local clock is not the grandmaster clock.
+
+.. Check that the expected `master offset` value corresponding to the value in `/var/run/ptp4l.0.config` is found in the `linuxptp-daemon-container` log:
++
+[source,terminal]
+----
+$ oc logs $PTP_POD_NAME -n openshift-ptp -c linuxptp-daemon-container
+----
++
+.Example output
+[source,terminal]
+----
+phc2sys[56020.341]: [ptp4l.1.config] CLOCK_REALTIME phc offset  -1731092 s2 freq -1546242 delay    497
+ptp4l[56020.390]: [ptp4l.1.config] master offset         -2 s2 freq   -5863 path delay       541
+ptp4l[56020.390]: [ptp4l.0.config] master offset         -8 s2 freq  -10699 path delay       533
+----
+
+. Check that the SR-IOV configuration is correct by running the following commands:
+
+.. Check that the `disableDrain` value in the `SriovOperatorConfig` resource is set to `true`:
++
+[source,terminal]
+----
+$ oc get sriovoperatorconfig -n openshift-sriov-network-operator default -o jsonpath="{.spec.disableDrain}{'\n'}"
+----
++
+.Example output
+[source,terminal]
+----
+true
+----
+
+.. Check that the `SriovNetworkNodeState` sync status is `Succeeded`:
++
+[source,terminal]
+----
+$ oc get SriovNetworkNodeStates -n openshift-sriov-network-operator -o jsonpath="{.items[*].status.syncStatus}"
```

Comment:
Please change this to: 

`oc get SriovNetworkNodeStates -n openshift-sriov-network-operator -o jsonpath="{.items[*].status.syncStatus}{'\n'}"`